eval/src/vespa/eval/eval/fast_sparse_map.h

```diff
@@ -66,7 +66,7 @@ class FastSparseMap
         Key() : hash(0) {}
         Key(uint64_t hash_in)
             : hash(hash_in) {}
-    } __attribute__((packed));
+    } __attribute__((aligned(4),packed));
```

Comment:
I am not sure if attributes are order sensitive, but we should switch them just to be safe.